frontend/packages/dev-console/src/components/health-checks/AddHealthCheckPage.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { RouteComponentProps } from 'react-router-dom';
+import { PageHeading, FirehoseResource, Firehose } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ServiceModel } from '@console/knative-plugin';
+import AddHealthCheckWrapper from './AddHealthCheckWrapper';
+
+type AddHealthCheckProps = RouteComponentProps<{
+  ns: string;
+  kind: string;
+  name: string;
+  containerName: string;
+}>;
+
+const AddHealthCheckPage: React.FC<AddHealthCheckProps> = ({ match }) => {
+  const { ns, kind, name, containerName } = match.params;
+  const resource: FirehoseResource[] = [
+    ...(kind === ServiceModel.kind
+      ? [
+          {
+            kind: referenceForModel(ServiceModel),
+            namespace: ns,
+            isList: false,
+            name,
+            prop: 'resource',
+          },
+        ]
+      : [
+          {
+            kind,
+            namespace: ns,
+            isList: false,
+            name,
+            prop: 'resource',
+          },
+        ]),
+  ];
+  return (
+    <>
+      <Helmet>
+        <title>Health Checks</title>
+      </Helmet>
+      <div>
+        <PageHeading title="Health Checks" />
+        <Firehose resources={resource}>
```

Comment:
Not a blocker for sure, but you can use `useK8sWatchResources` hook instead of `Firehose`. That way you won't need that extra wrapper component.